.github/workflows/lint.yml

```diff
@@ -27,4 +27,4 @@ jobs:
       - name: Swift Lint ClientRuntime
         uses: norio-nomura/action-swiftlint@3.1.0
```

Comment:
Latest version of this action is `3.2.1`... it's still not up to date, but maybe it gets us a more recent SwiftLint?

Reply:
https://github.com/norio-nomura/action-swiftlint/tree/3.2.1

Reply:
Good idea, should be updated now!